frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/DeleteModal/DeleteModal.test.tsx

```diff
@@ -54,15 +54,15 @@ describe('DeleteModal', () => {
 
     const mockDelete = jest.fn().mockImplementation(() => Promise.resolve());
 
-    render(null, { resetRepoChanges: mockDelete });
+    renderDeleteModal({}, { resetRepoChanges: mockDelete });
```

Comment:
I was a bit unclear about my intention in the previous comment. The reason I considered an object instead of null was to avoid sending parameters in the correct order and having to pass in null for you to be able to send in argument number two. If the argument to `renderDeleteModal ` is an object, then you can pass in only the properties that are needed and omit the rest. As the example below illustrates:

```suggestion
    renderDeleteModal({ queries: { resetRepoChanges: mockDelete }});
```

Reply:
Aaah okey. I solved it in another way. As the render methods first argument ```props``` is never used, there is no point to have it in there at the moment. If it for some reason might be needed in the future, it can be added when it is needed.